scripts/core/effects/add_item.py

```diff
@@ -0,0 +1,6 @@
+from scripts.core.effect import Effect
+
+
+class AddItemEffect(Effect):
```

Comment:
Are you thinking this would be an effect that add's an item? (which might seem a somewhat obvious question!)

Reply:
Yeah.  Its for one of the signature items which adds gold when moving to a new room.  https://github.com/Snayff/nqp2/issues/201